modules/kueue-providing-user-permissions.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * ai_workloads/kueue/monitoring-pending-workloads.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="providing-user-permissions_{context}"]
+= Providing user permissions
+
+[role=”_abstract”]
+You can configure role-based access control (RBAC) objects for the users of your {kueue-name} deployment. These objects determine which types of users can create which types of {kueue-name} objects.  
+
+You need to provide permissions to the users that require access to the specific APIs.
+
+* If the user needs access to the pending workloads of ClusterQueue, a ClusterRoleBinding needs to be created referencing the ClusterRole `kueue-batch-admin-role`.
+
+* If the user needs access to the pending workloads of LocalQueue, a RoleBinding needs to be created referencing the ClusterRole `kueue-batch-user-role`.
```

Comment:
LocalQueue and RoleBinding are objects and need ` `.

Reply:
Changed.